### PR TITLE
Clean up the bin/ directory, moving to bin/oneoff/ or deleting.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Further work remains, e.g., moving appropriate files from bin/ to bin/cron/ (updating the crontab.erb appropriately).

### CODE REVIEW

File: bin/grant-permission [hunk 1]
```diff
@@ -1,48 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (ashercodeorg) [inline bin/grant-permission:1]: This functionality is offered through the code.org/private/privileges UI.

File: bin/create-hoc-survery-user-id [hunk 2]
```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (ashercodeorg) [inline bin/create-hoc-survery-user-id:1]: Is this ever used?

Comment (aoby) [inline bin/create-hoc-survery-user-id:1]: no. I don't know anything about it.

**Comment by aoby** [comment]
LGTM

**Pull request merged**